packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -67,6 +67,18 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 			<div className="block-editor-block-inspector">
 				<MultiSelectionInspector />
 				<InspectorControls.Slot />
+				<InspectorControls.Slot
+					__experimentalGroup="typography"
+					label={ __( 'Typography' ) }
+				/>
+				<InspectorControls.Slot
```

Comment:
Is here a good opportunity to flip Dimensions and Border panels according to the order defined in https://github.com/WordPress/gutenberg/issues/35541? (Dimensions before Border)

Reply:
Absolutely. Done ✅